rs/ic_os/network/mac_address/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+load("@rules_rust//rust:defs.bzl", "rust_library", "rust_test")
+
+package(default_visibility = ["//rs:ic-os-pkg"])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "//rs/ic_os/utils",
+    "@crate_index//:anyhow",
+    "@crate_index//:hex",
+    "@crate_index//:regex",
+    "@crate_index//:serde",
+    "@crate_index//:sha2",
+]
+
+rust_library(
```

Comment:
We (later) should evaluate using [`deterministic_ips`](https://github.com/dfinity/ic/tree/4ddd8cfc1b7086d05299b0193fb65e2c9d03aed2/rs/ic_os/dev_test_tools/deterministic_ips) for this instead, or pick out the best parts of both as well as what already exists in the Rust ecosystem. Moving this into a crate of its own I think is a good first step.